flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -661,8 +690,14 @@ static bool msp_receive_byte(struct msp_bridge *m, uint8_t b)
  */
 static int32_t uavoMSPBridgeStart(void)
 {
-	if (!module_enabled)
+	if (!module_enabled) {
+		// give port to telemetry if it doesn't have one
+		// stops board getting stuck in condition where it can't be connected to gcs
+		if(!PIOS_COM_TELEM_RF)
```

Comment:
That's true but I would like to avoid target-specific code where possible. Since we compile this module on all targets (I think?) I don't think it's ever likely to be an issue.